media/moviResources.md

```diff
@@ -11,63 +11,78 @@ Such a resource contains the following components:
 * an index, describing the following data entries, and
 * the data entries.
 
-**MOVI Header** (256 bytes)
+**MOVI Header** (`MovieHeader` struct)
```

Comment:
Add the size of the struct again - 

    **MOVI Header** (`MovieHeader`struct) (256 bytes)